### ISSUE TITLE
wrong time, timezone and duration?

### ISSUE THREAD

**Issue by germebl** [H2-invent/jitsi-admin#521]
**Describe the bug**
Users getting displayed when entering the join-url an error like this: "Der Beitritt ist nur von 11.04.2024 12:40 UTC bis 11.04.2024 14:10 UTC möglich." while the time for the meeting is set to "13:10 – 14:10" (one hour duration set). So there is a beauty mistake: I'm not living in UTC so its the wrong timezone and further the time itself is absolutely wrong.

So i actually tested to change the php timezone to Europe/Berlin, cause all of our servers using UTC, but thats not affecting anything. Changing the timezone of the system is not a way to go.

**To Reproduce**
1. Create a new Meeting with a one hour duration from a Europe/Berlin TZ
2. To set the date and time use the calendar-widget from the start-time setting.
3. Disable Invite-Link via E-Mail and use the Invite Link after creation
4. Now you see a wrong duration, timezone and time.

**Branch or Version you using**
https://github.com/H2-invent/jitsi-admin/releases/tag/0.79.7

**Expected behavior**
First of all: timezone settings - depending on that it would be nice if the set timezone gets used on the error message instead of UTC. Not including me, some type of people may not familiar with timezones in that way. Further fix the time duration issue.

Further a fixed time, timezone and duration.

**Screenshots**
![1](https://github.com/H2-invent/jitsi-admin/assets/129985242/0f545a20-0cd7-4094-afac-686580d5ce2b)
![2](https://github.com/H2-invent/jitsi-admin/assets/129985242/36299c14-428b-421b-a0a6-41193c26963f)

**Comment by holema** [comment]
Hello @germebl,

I've identified several issues that need addressing.

Firstly, the default timezone for all meetings is derived from the PHP timezone settings. If the timezone isn't explicitly adjusted when setting the meeting time, the default timezone is automatically selected. Additionally, all meetings are internally converted to UTC to ensure accurate time comparison with the current time.

Therefore, when users aren't logged in, the UTC time is displayed as the standard time, which is technically correct. However, we may need to explore alternative solutions.

Furthermore, if the timezone feature is disabled in the Jitsi admin (by setting allowTimeZoneSwitch=0), users will see the time according to the default PHP timezone.

Regarding your mention of a "completely wrong time," I'm unclear on what exactly you're referring to. Could you please provide further clarification?

Feel free to email me at entwicklung@h2-invent.com so we can investigate whether this issue stems from a bug in your installation or within the project itself.

**Comment by germebl** [comment]
Hey @holema ,

the issue is solved from our side. The issue where, that the PHP timezone of PHP-FPM was correctly set to Europe/Berlin, but PHP-CLI still got the default timezone which is UTC. After setting CLI to Europe/Berlin the issue where fixed.

I said "completely wrong time" cause we created a meeting which should start from 1:10pm to 2:10pm but the error showed its available from 12:45pm to 2:10pm. We dont know where the half hour came from in front of the time we set. But it also fixed with the correct timezone.
